xarray/core/dataset.py

```diff
@@ -3463,7 +3463,7 @@ def drop(
     ) -> "Dataset":
         ...
 
-    def drop(self, labels, dim=None, *, errors="raise"):  # noqa: F811
+    def drop(self, labels=None, dim=None, *, errors="raise", **labels_kwargs):
```

Comment:
```suggestion
    def drop(self, labels=None, dim=None, *, errors="raise", **labels_kwargs):  # noqa: F811
```

Reply:
Hopefully this doesn't break black - may need to ensure the `noqa` is in the right place if black reformats

Reply:
Thanks. I had to move this comment slightly after using `black`, but it worked.